nav2_collision_monitor/src/collision_monitor_node.cpp

```diff
@@ -480,6 +495,16 @@ void CollisionMonitor::printAction(
   }
 }
 
+void CollisionMonitor::publishActionState(const Action & robot_action) const
```

Comment:
I'd prefer if this as combined with the print function into a `notifyActionState` method logging + publishing